lib/data_cite/metadata/base.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module DataCite
+  module Metadata
+    class Base
+      class Error < StandardError; end
+      class ValidationError < Error; end
```

Comment:
This is nit-picky, so take it or leave it, but could you just move the class definition to Metadata? DataCite::Metadata::Base::Error seems wordy, and having DataCite::Metadata::Error matches up with DataCite::Client::Error.

Reply:
That's a good idea, and I think I'll do that. That's the way it was before, but I must have lost it in the refactor madness